infrastructure/testdriver/set_permission.https.html

```diff
@@ -1,17 +1,23 @@
 <!DOCTYPE html>
-<meta charset="utf-8">
+<meta charset="utf-8" />
```

Comment:
This seems unrelated?

Reply:
No, ran prettier over it… it adds the “ />” garbage.